server/__tests__/formatters/socketHandlerFormatter.js

```diff
@@ -0,0 +1,37 @@
+import { formatOutput } from '../../formatters/socketHandlerFormatter';
+
+describe('server/formatters/socketHandlerFormatter', () => {
+  describe('formatOutput', () => {
+    it('should be a function', () => {
+      expect(typeof formatOutput).toEqual('function');
+    });
+
+    it('throws an error when both props of `error` and `data` are passed or both are not passed to the function.', () => {
```

Comment:
@duyoji
What does this data express?
Why do we need this function?

Reply:
I explained the reason fo this feature to Ryuta directly.